diff --git a/preview-src/driver.adoc b/preview-src/driver.adoc
index 7566dd4..c5beb27 100644
--- a/preview-src/driver.adoc
+++ b/preview-src/driver.adoc
@@ -1,7 +1,7 @@
 = Neo4j & Python
 :page-theme: docs
 :page-role: code-walkthrough
-:page-chatbot: http://localhost:3000/
+:page-chatbot: http://localhost:3000
 :!toc:
 
 
diff --git a/src/partials/footer-scripts.hbs b/src/partials/footer-scripts.hbs
index 53b37e4..c70e1ef 100644
--- a/src/partials/footer-scripts.hbs
+++ b/src/partials/footer-scripts.hbs
@@ -30,7 +30,7 @@
 
 {{#if page.attributes.chatbot}}
 <div id="docs_chatbot"></div>
-<script type="module" crossorigin src="{{{ page.attributes.chatbot }}}index.js"></script>
+<script type="module" crossorigin src="{{{ page.attributes.chatbot }}}/index.js"></script>
 {{/if}}
 
 <div id="neo4j-algolia-search-v2"></div>
diff --git a/src/partials/head-styles.hbs b/src/partials/head-styles.hbs
index 0366c56..cb256e3 100644
--- a/src/partials/head-styles.hbs
+++ b/src/partials/head-styles.hbs
@@ -21,5 +21,5 @@
 
 
 {{#if page.attributes.chatbot}}
-<link rel="stylesheet" crossorigin href="{{{ page.attributes.chatbot }}}index.css">
+<link rel="stylesheet" crossorigin href="{{{ page.attributes.chatbot }}}/index.css">
 {{/if}}
\ No newline at end of file
